packages/plans-grid/package.json

```diff
@@ -0,0 +1,52 @@
+{
+	"name": "@automattic/plans-grid",
+	"version": "1.0.0-alpha.0",
+	"description": "WordPress.com Plans Grid UI component",
+	"homepage": "https://github.com/Automattic/wp-calypso",
+	"license": "GPL-2.0-or-later",
+	"author": "Automattic Inc.",
+	"main": "dist/cjs/index.js",
+	"module": "dist/esm/index.js",
+	"sideEffects": [
+		"*.css",
+		"*.scss"
+	],
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/Automattic/wp-calypso.git",
+		"directory": "packages/plans-grid"
+	},
+	"publishConfig": {
+		"access": "public"
+	},
+	"bugs": {
+		"url": "https://github.com/Automattic/wp-calypso/issues"
+	},
+	"types": "dist/types",
+	"scripts": {
+		"clean": "check-npm-client && npx rimraf dist ../../.tsc-cache/packages__plans-grid*",
+		"prepare": "check-npm-client && tsc --project ./tsconfig.json && tsc --project ./tsconfig-cjs.json && copy-assets",
+		"prepublish": "check-npm-client && yarn run clean",
+		"watch": "check-npm-client && tsc --project ./tsconfig.json --watch"
+	},
+	"dependencies": {
+		"@automattic/data-stores": "^1.0.0-alpha.1",
+		"@automattic/react-i18n": "^1.0.0-alpha.0",
+		"@wordpress/base-styles": "^1.8.0",
```

Comment:
One more thing (it's my bad I didn't include `-D` in my recommendation). [According to base-style docs](https://developer.wordpress.org/block-editor/packages/packages-base-styles/#:~:text=%20--save-dev), they only need to be a devDep. Probably because they're compiled at build time.